src/qibocal/auto/runcard.py

```diff
@@ -52,4 +54,8 @@ def load(cls, card: Union[dict, Path]):
             if isinstance(card, Path)
             else card
         )
-        return cls(**content)
+        return cls(
+            actions=content["actions"],
+            qubits=content["qubits"] if "qubits" in content else [],
+            format=content["format"] if "format" in content else None,
+        )
```

Comment:
Since you're creating `content` in this scope, you could also just modify it:
```suggestion
        content.setdefault("qubits", [])
        content.setdefault("format")
        return cls(**content)
```

Reply:
In principle, this way of doing things would fail if there is some unrecognized field in the `content`.
I consider this a feature: unknown fields should not be present in a runcard.